packages/pytest-simcore/src/pytest_simcore/docker_registry.py

```diff
@@ -230,7 +230,7 @@ def jupyter_service(docker_registry: str, node_meta_schema: Dict) -> Dict[str, s
     )
 
 
-DY_STATIC_FILE_SERVER_VERSION = "1.0.5"
+DY_STATIC_FILE_SERVER_VERSION = "2.0.0"

```

Comment:
We could go for a fixture, but I see no additional value in testing with different images. Since this is mainly a validation image for new features. I think this works well enough. Will put a fixture in.